### DISCUSSION TITLE
File Transfers Between Local Computer and v86

### DISCUSSION THREAD

**Discussion by chaosAD** [copy/v86#917] [Ideas]
Any chance to include capability to easily move files between local computer and the v86 by drag-and-drop?

**Comment by spetterman66** [comment]
it's possible with the 9p file system (linux4 and buildroot profiles)

**Reply by ZXMushroom63** [reply]
is it possible without 9p?
I'm just trying to read files from my ubuntu image, and having a lot of trouble.

**Reply by ZXMushroom63** [reply]
update: I found a method using the serial port ttyS0
